eng/common/scripts/Verify-Links.ps1

```diff
@@ -124,28 +124,34 @@ function CheckLink ([System.Uri]$linkUri)
     }
   }
   else {
+     $headRequestSucceeded = $true
     try {
-      $response = Invoke-WebRequest -Uri $linkUri
-      $statusCode = $response.StatusCode
-      if ($statusCode -ne 200) {
-        Write-Host "[$statusCode] while requesting $linkUri"
-      }
+      # Attempt HEAD request first
+      $response = Invoke-WebRequest -Uri $linkUri -Method HEAD
     }
     catch {
-      $statusCode = $_.Exception.Response.StatusCode.value__
-
-      if ($statusCode -in $errorStatusCodes) {
-        LogWarning "[$statusCode] broken link $linkUri"
-        $script:badLinks += $linkUri 
+      $headRequestSucceeded = $false
+      }
+    if (!$headRequestSucceeded) {
+      # Attempt a GET request if the HEAD request failed.
+      $response = Invoke-WebRequest -Uri $linkUri -Method GET
```

Comment:
this call in particular still needs to be inside a try statement. Please make this code match what I merged into the azure-sdk repo.